src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -504,6 +504,7 @@ protected virtual void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         /// <summary>
         /// Contains event arguments for the FocusChangeRequested event.
         /// </summary>
+        [Obsolete("Deprecated in API9; Will be removed in API12.")]
```

Comment:
According to our deprecation policy it should be "Deprecated in API9; Will be removed in API11." instead. Also you should suggest the alternative if there is any.

Reply:
Fixed! thank you~